code/obj/artifacts/artifact_machines/plant_helper.dm

```diff
@@ -32,21 +32,54 @@
 		if (..())
 			return
 		for (var/obj/machinery/plantpot/P in range(O,src.field_radius))
+			var/r = 0
+			var/g = 0
+			var/b = 0
+			var/total = 0
 			var/datum/plant/growing = P.current
 			for (var/X in src.helpers)
 				if (X == "water")
 					var/wateramt = P.reagents.get_reagent_amount("water")
 					if(wateramt > 200)
 						P.reagents.remove_reagent("water", 1)
+						r += 255
+						total++
 					if(wateramt < 100)
 						P.reagents.add_reagent("water", 1)
+						b += 255
+						total++
 				if (X == "growth" && growing)
 					P.growth++
+					g += 255
+					total++
 				if (X == "health" && growing)
 					P.health++
+					r += 255
+					g += 128
+					b += 128
+					total++
 				if (X == "weedkiller" && growing)
 					if (growing.growthmode == "weed")
 						P.health -= 3
+						r += 255
+						total++
 				if (X == "mutation" && growing)
 					if (prob(8))
 						P.HYPmutateplant()
+						total = INFINITY
+				if(total)
+					SPAWN_DBG(0)
+						var/lineColor = rgb(r/total, g/total, b/total)
+						var/datum/lineResult/R = drawLine(get_turf(O), P, "smooth", "smoothCap", getCrossed = 0)
+						var/globalImageKey = "linetest[rand(0,INFINITY)]"
+						R.lineImage.color = lineColor
+						R.lineImage.alpha = 0
+						R.lineImage.plane = PLANE_SELFILLUM
+						R.lineImage.blend_mode = BLEND_ADD
+						addGlobalImage(R.lineImage, globalImageKey)
+						animate(R.lineImage, alpha = 64, time = 1 SECOND)
+						sleep(1 SECONDS)
+						animate(R.lineImage, alpha = 0, time = 1 SECOND)
+						sleep(1 SECONDS)
```

Comment:
can't we use animate timers here instead or something

Reply:
Oh, yeah, I was thinking that while testing, but then I forgot about it! I'll try to do it with timers!